docker-entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -ex
+
+echo "Running in $PWD"
+
+mkdir -p /opt/hoover/snoop/testsuite/volumes/snoop-pg/data
```

Comment:
We don't want to know / control / think about any database data directory in this service.